### ISSUE TITLE
Notification updates

### ISSUE THREAD

**Issue by DoganM95** [binwiederhier/ntfy#1182]
:bulb: **Idea**
There should really be a way to update an existing notification. I tried creating one, even by passing the header "Id: something-unique-123" so that the subsequent call of this post request would update the old one, but it just creates a new identical one, leaving many duplicates in the notifications.

:computer: **Target components**
Server and apps

**Comment by Rycoh99** [comment]
+1, I'd love this

**Comment by 6r1d** [comment]
+1, a very useful idea

**Comment by baudneo** [comment]
Agree, this would make frigate and other NVR notifications much better on Android. Instead of getting multiple notifications about the same event (first the label of the detected object with a low res thumbnail and then a generated LLM notification with a high quality snapshot) you can update the low res one with the high res image and the generated response.

**Comment by binwiederhier** [comment]
This is a duplicate of https://github.com/binwiederhier/ntfy/issues/303
